packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts

```diff
@@ -0,0 +1,62 @@
+import type { IGLTFExporterExtensionV2 } from "../glTFExporterExtension";
+import { _Exporter } from "../glTFExporter";
+import type { Material } from 'core/Materials';

```

Comment:
should be import type { Material } from 'core/Materials/material';

Reply:
Updated